cmds/replay.c

```diff
@@ -589,7 +589,7 @@ void get_argspec_string(struct uftrace_task_reader *task,
 			if (last_ch == '\n')
 				newline = 1;
 
-			str = xmalloc(slen + newline + 1);
+			str = xmalloc(slen + newline + sizeof(null_str));
```

Comment:
Hmm.. I don't like it.  Actually I found two other concerns.  First, checking newline character is not needed anymore since we use `print_escape_char()`.  And the `null_str` check is just for older data formats and I think you'd better checking `slen == 4` before calling `memcmp` instead.

Reply:
much better, I modified it.